components/classes/ClassStudentList.tsx

```diff
@@ -60,7 +60,7 @@ const ClassStudentList: StylableFC<{
         // Download the file
         window.location.href = URL.createObjectURL(mergedVCard);
 
-        va.track("Save Class VCards", { number: `M.${classNumber}` });
+        va.track("Save Class VCards", { number: `M.${classroom.number}` });
```

Comment:
Restructuring code this way makes it much easier to understand and use in the future.